blog/poc3.md

```diff
@@ -0,0 +1,120 @@
+ communications tool ([Onyx](https://blog.mainframe.com/mainframe-pre-alpha-release-fca532317111)) and their web3 SDK, and [Status](http://status.im) have expressed interest in building it into their mobile chat.
+
+## Mutability
+
+Hash addressing is immutable in the strong sense that you cannot even express mutable content: "changing the content changes the hash". Users of the web, however, are accustomed to mutable resources: when visiting URL-s we expect to see the most up to date version of the 'site'. Mutable resources on Swarm are made possible by the Ethereum Name Service (ENS)](http://swarm-guide.readthedocs.io/en/latest/usage.html#Ethereum) and [*Mutable Resource Updates (MRU)*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Mutable).
+The ENS is a smart contract on the Ethereum blockchain which enables domain owners to register a content reference to their domain.
+Using ENS for domain name resolution, the URL scheme provides
+content retrieval based on mnemonic (or branded) names, much like the DNS of the world wide web, but without servers.
+Mutable Resource Update is, on the other hand, an off-chain solution for communicating updates to a resource, it offers cheaper and potentially faster updates than ENS, yet the updates can be consolidated on ENS by any third party willing to pay for the transaction.
+
+Here is how it works: a publisher creates a mutable resource at time `t` for the name "my daily tweets", that she intends to update every day once (note this in itself is not a promise). By hashing together `ID := SHA3(publisher_pubkey, name, start, period)`, we obtain a so called metadata chunk.
+A publisher creates a new version of a mutable resource by serialising `(name, start, period, version)` and signing it.
+This simple system is surprisingly powerful and is envisioned to be at the core of implementing group chat, shared FUSE state, and [poor man's consensus](https://www.sharelatex.com/1452913241cqmzrpfpjkym) (soft channel deposit).
+
+MRU is an experimental feature in current POC3 testnet and is still  undergoing changes.
+
+## FUSE support
+
+We mentioned that thanks to manifests, directory trees and collections represented by manifests can be mapped uniquely. This does not only allow upload and download into directory but also thanks to [FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to mount a Swarm manifest as a local file system (only available on Linux and Mac).
+Due to the fact that Swarm retrieval is on-demand, Swarm FUSE mounts are readable with no latency. Swarm FUSE supports file system write operations (creating, appending, updating files) by maintaining the collection's Swarm hash as an internal state.
+Combining FUSE with MRU, on each change we can issue a mutable resource update, this makes it possible to think of FUSE to sync your entire home folder between devices, virtually serving as the backend to a decentralised drop box.
```

Comment:
Dropbox is one word, and I would use the term "Dropbox-like functionality",  because dropbox is an actual company/brand/product